src/api/app/models/package.rb

```diff
@@ -151,13 +151,23 @@ def self.internal_get_project(project)
     prj
   end
 
+  def self.normalize_package_name(name)
+    # exception for package names used to have a collon
```

Comment:
What are you trying to do here? There are packages in that have a colon in the name? How can this be?